src/openkit-shared/Protocol/HTTPClientHttpClient.cs

```diff
@@ -23,11 +23,13 @@ namespace Dynatrace.OpenKit.Protocol
 #if (!NET40 && !NET35)
 
     public class HTTPClientHttpClient : HTTPClient
-    {
+    {

```

Comment:
as discussed before, the cascaded #if preprocessor declarations are hard to read.